app/src/main/res/values-night/colors.xml

```diff
@@ -180,13 +180,13 @@
 
     <!-- Search Widget -->
     <color name="search_widget_background">@color/inset_normal_theme</color>
-    <color name="search_widget_mic_fill_color">@color/primary_text_dark_theme</color>
-    <color name="search_widget_text">@color/primary_text_dark_theme</color>
+    <color name="search_widget_mic_fill_color">@color/photonLightGrey05</color>
```

Comment:
It's unclear to me what design token should be used for `search_widget_mic_fill_color` and `mozac_ui_tabcounter_default_tint`. So, I am leaving it as `photonLightGrey05`